### ISSUE TITLE
Right-click context relative to nested items

### ISSUE THREAD

**Issue by broderickhyman** [erdomke/InnovatorAdmin#205]
Looking to add the ability to right-click a nested item, for instance an item in a Relationships tag or a related_id tag, and have the context menu reflect that item. Currently it only reflects the outer-most item.
I don't mind making the change, but need some direction. I think it has something to do with AmlSimpleEditorHelper.GetCurrentItem.
@erdomke

**Comment by erdomke** [comment]
@broderickhyman Yeah, the code is a bit of a mess, and I would find that feature useful too. After a quick gander, I think the relevant code is in the [AmlSimpleEditorHelper.GetCurrentItem](https://github.com/erdomke/InnovatorAdmin/blob/ffa885886a01c6518858f0b2e16966724a2813a5/InnovatorAdmin/Editor/Aml/AmlSimpleEditorHelper.cs#L125) method.
